paasta_tools/long_running_service_tools.py

```diff
@@ -314,6 +314,9 @@ def is_autoscaling_enabled(self) -> bool:
     def get_max_instances(self) -> Optional[int]:
         return self.config_dict.get("max_instances", None)
 
+    def get_instances_from_config(self) -> Optional[int]:
+        return self.config_dict.get("instances", None)
```

Comment:
Probably worth adding a docstring that explains why this method exists and how it differs from `get_instances`.